### PR TITLE
Add tutorial for implementation Live betting in the existing Prematch dApp

### PR DESCRIPTION
**Pull request by on47sky**

### CODE REVIEW

File: pages/hub/apps/guides/advanced/live-tutorial/data-retrieval.mdx [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+This allows you to reuse game/sport queries for both Prematch and Live requests.
+
+Here's an example of a universal hook to retrieve events from both Prematch and Live feeds:
+
+<Tabs items={[ 'useSportsAndEvents.ts', 'query.ts' ]}>
+<Tabs.Tab>
+```ts {6,11,17-19,22-23,29} copy
+import { universalGamesQuery } from './query'
+
+const useSportsAndEvents = (props) => {
```
Comment (CleanBread) [inline pages/hub/apps/guides/advanced/live-tutorial/data-retrieval.mdx:33]: why is it sports and events if you get only games?

File: pages/hub/apps/guides/advanced/live-tutorial/data-retrieval.mdx [hunk 2]
```diff
@@ -0,0 +1,313 @@
...
+</Callout>
+
+## Load markets
+
+Because of the difference between Live and Prematch `condition` entities,
+it's not possible to reuse the Prematch query. You'll need to create a new one.
+
+Take a look at the source code of the SDK's [useGameMarkets](https://github.com/Azuro-protocol/sdk/blob/v3.1.0/src/hooks/useGameMarkets.ts) hook.
+It fetches conditions from the Live feed when the game is live, and Prematch conditions when the game hasn't started yet.
+To streamline the handling of both prematch and live conditions in the SDK, we're modifying these entities to have the same fields.
```
Comment (CleanBread) [inline pages/hub/apps/guides/advanced/live-tutorial/data-retrieval.mdx:122]: actually useConditions does that, useGameMarkets combines conditions in markets

**Review by Avvrik — APPROVED** [review]

**Review by CleanBread — APPROVED** [review]

File: pages/hub/apps/guides/advanced/live-tutorial/data-retrieval.mdx (hunk 1 above)
Comment (on47sky) [reply pages/hub/apps/guides/advanced/live-tutorial/data-retrieval.mdx]: https://github.com/Azuro-protocol/gem-docs/pull/155/commits/f9e8ff61f1172594f6046f4b2fbbfab8f0422e1f

File: pages/hub/apps/guides/advanced/live-tutorial/data-retrieval.mdx (hunk 2 above)
Comment (on47sky) [reply pages/hub/apps/guides/advanced/live-tutorial/data-retrieval.mdx]: https://github.com/Azuro-protocol/gem-docs/pull/155/commits/f9e8ff61f1172594f6046f4b2fbbfab8f0422e1f

**Pull request merged**